functions.php

```diff
@@ -444,22 +444,22 @@ function twentytwenty_block_editor_settings() {
 	add_theme_support(
 		'editor-font-sizes',
 		array(
-			array(
-				'name'      => _x( 'Small', 'Name of the small font size in the block editor', 'twentytwenty' ),
-				'shortName' => _x( 'S', 'Short name of the small font size in the block editor.', 'twentytwenty' ),
-				'size'      => 18,
-				'slug'      => 'small',
-			),
 			array(
 				'name'      => _x( 'Regular', 'Name of the regular font size in the block editor', 'twentytwenty' ),
 				'shortName' => _x( 'M', 'Short name of the regular font size in the block editor.', 'twentytwenty' ),
 				'size'      => 21,
 				'slug'      => 'regular',
 			),
+			array(
+				'name'      => _x( 'Small', 'Name of the small font size in the block editor', 'twentytwenty' ),
+				'shortName' => _x( 'S', 'Short name of the small font size in the block editor.', 'twentytwenty' ),
+				'size'      => 18,
+				'slug'      => 'small',
+			),
 			array(
 				'name'      => _x( 'Large', 'Name of the large font size in the block editor', 'twentytwenty' ),
 				'shortName' => _x( 'L', 'Short name of the large font size in the block editor.', 'twentytwenty' ),
-				'size'      => 24,
+				'size'      => 26.25,
```

Comment:
Do we need to make the large font size even larger?

Reply:
This new size makes the size in editor view match the size on the frontend. Frontend calculates to 26.25px